dockerfiles/jupyter/README.ipynb

```diff
@@ -77,7 +77,11 @@
     "* [Pipelines](https://github.com/kubeflow/pipelines)\n",
     "* [Prometheus stack](https://github.com/prometheus-community/helm-charts)\n",
     "  - Prometheus\n",
-    "  - Grafana"
+    "  - Grafana\n",
+    "* MLRun Model monitoring: requires the installation of the following two applications. See more details in [Selecting the streaming and TSDB platforms](https://docs.mlrun.org/en/stable/model-monitoring/index.html#selecting-the-streaming-and-tsdb-platforms).\n",
+    "    * [TDengine](https://taosdata.github.io/TDengine-Operator/en/2.2-tdengine-with-helm.html) (3.3.2.0) \n",
+    "    * [Kafka](https://github.com/bitnami/charts/tree/main/bitnami/kafka) (3.2.3)\n",
+    "    > In MLRun 0.8.0, `TDengine` and `Kafka` are installed by default. If you are using an older chart version, please install those components manually along with the MLRun chart."
```

Comment:
" > In MLRun 0.8.0, `TDengine` and `Kafka` are installed by default. If you are using an older chart version, install those components manually along with the MLRun chart."